packages/sites/src/index.ts

```diff
@@ -58,3 +58,4 @@ export * from "./get-data-for-site-item";
 export * from "./get-site-by-id";
 export * from "./is-site";
 export * from "./get-members";
+export * from "./_ensure-telemetry";
```

Comment:
should this be exported here? It appears this is a "private" fn that we only export for ease of testing.

Reply:
I only added it b/c I noticed other private method & migration module exports, doesn't need exported, so happy to remove

Reply:
Scratch that, deleting the export statement breaks tests that try to mock out this migration